examples/cpp/CMakeLists_Standalone.txt

```diff
@@ -2,9 +2,19 @@ cmake_minimum_required(VERSION 3.10)
 project(example)
 
 include(ExternalProject)
+include(FetchContent)
+
+# Function to fetch latest release version from GitHub
+function(fetch_latest_release owner_name repo_name out_variable)
+    file(DOWNLOAD "https://api.github.com/repos/${owner_name}/${repo_name}/releases/latest" "${CMAKE_BINARY_DIR}/latest_release.json" TLS_VERIFY ON)
```

Comment:
Use CMAKE_BUILD_DIR over CMAKE_BINARY_DIR, otherwise the artifact folder will also contain the json file as well. it's a temporary file, it should exist in a temporary folder.

Reply:
can't use CMAKE_BINARY_DIR,
tried CMAKE_CACHEFILE_DIR, but to make it works you need to run twice (something related to permissions)

was unable to find CMAKE_BUILD_DIR
https://gitlab.kitware.com/cmake/community/-/wikis/doc/cmake/Useful-Variables